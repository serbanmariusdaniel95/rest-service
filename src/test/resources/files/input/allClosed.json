{
  "displayed_what" : "Casa Ferlin",
  "displayed_where" : "Stampfenbachstrasse 38",
  "opening_hours" : {
  }
}