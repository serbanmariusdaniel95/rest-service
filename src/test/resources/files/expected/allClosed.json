{
  "name" : "Casa Ferlin",
  "address" : "Stampfenbachstrasse 38",
  "openingHours" : {
    "Monday - Sunday": [
      "closed"
    ]
  }
}